db/migrate/20240809143631_add_fields_to_gmail_messages.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AddFieldsToGmailMessages < ActiveRecord::Migration[7.1]
+  def change
+    safety_assured do
+      add_column :gmail_messages, :body, :text
+      add_column :gmail_messages, :from, :string
+      add_column :gmail_messages, :to, :string
+      add_column :gmail_messages, :subject, :string
+      add_column :gmail_messages, :received_at, :datetime
```

Comment:
Good point from standup today, let's encrypt these fields by default? Should be straightforward as  adding `encrypts` in the model: https://guides.rubyonrails.org/active_record_encryption.html#declaration-of-encrypted-attributes